src/middleware/validations.ts

```diff
@@ -107,27 +107,58 @@ export default class Validations {
 	}
 
 	static updateProfileValidationRules() {
-		return [body('aboutMe', 'About me text is required!').exists()];
+		return [body('aboutMe', 'About me text is required!').trim()];
 	}
 
 	static updateAccountDetailsRules() {
 		return [
-			body('fName', 'First name is required!').exists(),
-			body('lName', 'Last name is required!').exists(),
+			body('fName', 'First name is required!').trim().isLength({ min: 1 }).isString(),
+			body('lName', 'Last name is required!').trim().isLength({ min: 1 }).isString(),
 		];
 	}
 
+	private static readonly AGENCY_PHONE_NUMBER_REGEX =
+		/^\(?([0-9]{3})\)?([ .-]?)([0-9]{3})\2([0-9]{4})/;
+
```

Comment:
@stacysealky of course, this is the new code change, please take a look and check if it's correct yet
```suggestion
	private static readonly AGENCY_PHONE_NUMBER_REGEX =
		/^[(]?[2-9][0-9]{2}[)]?[-\s]?[0-9]{3}[-\s]?[0-9]{4}$/;

```

Reply:
I actually would update it a bit further

```js
/^(\([2-9][0-9]{2}\)[-\s]|[2-9][0-9]{2}[-\s])?[0-9]{3}[-\s]?[0-9]{4}$/
```

this matches the following strings

`123-1222`
`(234)-123-1222`
`234-123-1222`
`(234) 123-1222`
`234 123-1222`

your solution would also match `(223-123-1222` `223)-123-1222` which is not correct I think